video/out/vo_gpu_next.c

```diff
@@ -889,6 +889,11 @@ static void apply_target_options(struct priv *p, struct pl_frame *target)
         target->color.hdr.max_luma = opts->target_peak;
     if (!target->color.hdr.min_luma)
         apply_target_contrast(p, &target->color);
+    if (opts->target_gamut) {
+        const struct pl_raw_primaries *prim;
+        prim = pl_raw_primaries_get(mp_prim_to_pl(opts->target_gamut));
+        target->color.hdr.prim = *prim;
```

Comment:
What if `target->color.primaries` is smaller than `target->color.hdr.prim`?

Reply:
Then the user is ~~stupid~~ unwise?

More seriously, you will just get hard clipping.

Reply:
I mean, I could add a `pl_primaries_clip` call but I don't really see the point.

Reply:
> Then the user is stupid unwise?

I guess we cannot babysit everyone. But maybe a short MP_WARN about this "unusual" configure would avoid some stupidness. 

> More seriously, you will just get hard clipping.

That what I recalled it would do.

Reply:
Updated to use `pl_primaries_clip` to ensure it fits inside the container